### DISCUSSION TITLE
Cameras failure when moving from scene to scene

### DISCUSSION THREAD

**Discussion by LoafPie** [ramokz/phantom-camera#322] [Question / Help Wanted]
If I start any scenes with CameraPhantoms it works perfectly. But if I change scenes through the code nothing works. I get warn messages about multiple CameraHost and I can see that I have two nodes in group CameraHost.

**Comment by ramokz** [comment]
Does updating to the latest version resolve this?
It should no longer rely on the Godot group system.
